core/geometry/src/curve/CurveCurveCloseApproachXY.ts

```diff
@@ -483,49 +509,53 @@ export class CurveCurveCloseApproachXY extends NullGeometryHandler {
     arc: Arc3d,
     reversed: boolean,
   ): void {
-    // To consider:
-    // 1) endpoint to endpoint or projection
-    // 2) true intersection
-    // 3) line parallel to arc tangent.
-    this.testAndRecordFractionalPairApproach(cpA, 0, 1, true, arc, 0, 1, false, reversed);
-    // Arc: X = C + cU + sV
-    // Line:  contains points A0,A1
-    // Arc point colinear with line if det (A0, A1, X) = 0
-    // with homogeneous xyw points and vectors.
-    // With equational X:   det (A0, A1, C) + c det (A0, A1,U) + s det (A0, A1, V) = 0.
+    // 1) arc endpoints to segment endpoints or arc endpoints projection to the segment
+    this.testAndRecordFractionalPairApproach(cpA, 0, 1, true, arc, 0, 1, true, reversed);
```

Comment:
Should be true, false. Accidentally changed to true,true.

Reply:
I tried to revert to "true, false", but that caused a test failure. Can you write a test that fails if "true, true" is passed in?